learn/dapps-list/dapp-directory.md

```diff
@@ -0,0 +1,299 @@
+---
+title: List your Dapp on the Moonbeam DApp Directory
+description: Follow this tutorial to learn how to list your Moonbeam or Moonriver project or update a current listing on the Moonbeam Foundation DApp Directory.
+---
+
+# How to List your Project on the Moonbeam DApp Directory
+
+## Introduction to The Moonbeam DApp Directory {: #introduction-to-state-of-the-dapps }
+
+The Moonbeam ecosystem comprises two distinct networks: Moonbeam and Moonriver. Each network has its own dedicated [DApp Directory](https://apps.moonbeam.network/moonbeam/app-dir){target=\_blank}, maintained by the Moonbeam Foundation. These directories categorize projects spanning from DeFi to NFTs to gaming, providing users with comprehensive access to diverse applications.
```

Comment:
```suggestion
The Moonbeam ecosystem comprises two distinct production networks: Moonbeam and Moonriver. Each network has its own dedicated [DApp Directory](https://apps.moonbeam.network/moonbeam/app-dir){target=\_blank}, maintained by the Moonbeam Foundation. These directories categorize projects spanning from DeFi to NFTs to gaming, providing users with comprehensive access to diverse applications.
```

Reply:
Or MainNet, whichever you prefer! just something to clarify so that people don't think that Moonriver is a testnet